src/components/home/ParsedInput.vue

```diff
@@ -134,12 +121,10 @@ export default Vue.extend({
       return 'error'
     },
     formatGenes() {
-      // this.loading = true
-      $getFindGenesWorker().postMessage({
-        init: false,
-        todo: 'findAllGenes',
-        userGenes: this.userGenes,
-      })
+      this.formattedGenes = new ParsedGenes()
+      this.formattedGenes.notFoundGenes = this.parsedGenes.notFoundGenes
+      this.formattedGenes.synonymFoundGenes = this.parsedGenes.synonymFoundGenes
+      this.formattedGenes.symbolFoundGenes = this.parsedGenes.symbolFoundGenes
```

Comment:
We can probably use this.parsedGenes directly now instead. This way we also get rid of the watch

Reply:
ignored this